.github/workflows/release-dev.yaml

```diff
@@ -168,6 +168,7 @@ jobs:
           - TestMultiNodeHAInstallation
           - TestMultiNodeAirgapHAInstallation
           - TestMultiNodeAirgapUpgradeSameK0s
+          - TestProxiedEnvironment
```

Comment:
this file needs the same build change as `.github/workflows/pull-request.yaml` (to include `K0S_BINARY_SOURCE_OVERRIDE=$(make print-PREVIOUS_K0S_BINARY_SOURCE_OVERRIDE)`)